libs/ml/include/ml/model/model.hpp

```diff
@@ -743,4 +366,5 @@ struct MapSerializer<ml::model::Model<TensorType>, D>
 };
```

Comment:
(optional) could static member methods of MapSerialiser be transferred to .cpp file too?

Reply:
Nope, it seems they won't be accessible from the outer scope

Reply:
I moved them to serialisers/ml_types.h instead